e2e/test/scenarios/embedding/embedding-dashboard.cy.spec.js

```diff
@@ -515,6 +515,7 @@ describe("scenarios > embedding > dashboard parameters", () => {
         isEmbed: true,
         logResults: true,
         downloadUrl: "/api/embed/dashboard/*/dashcard/*/card/*/csv*",
+        downloadMethod: "GET",
```

Comment:
This was needed because of https://github.com/metabase/metabase/pull/45240/commits/ce32b2ac9e14728ad7fa805c94dc429a9a528c12